source/common/router/scoped_config_impl.h

```diff
@@ -167,6 +167,8 @@ class ScopedRouteInfo {
       }
     }
   }
+  ~ScopedRouteInfo() = default;
```

Comment:
nit: for consistency -- a defaulted non virtual destructor tends to be considered superfluous.

Reply:
done